feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -43,6 +44,7 @@ internal class HomeViewModel @Inject constructor(
     private val getGeolocationPermissionRequestedSettingUseCase: GetGeolocationPermissionRequestedSettingUseCase,
     private val updateGeolocationPermissionRequestedSettingUseCase: UpdateGeolocationPermissionRequestedSettingUseCase,
     private val getTimerStateUseCase: GetTimerStateUseCase,
+    private val getAnimalTypeSettingsUseCase: GetAnimalTypeSettingsUseCase,
```

Comment:
refactor to single app settings usecase

Reply:
Good idea to combine all App settings in one UseCase, for my view, it's over-engineering to wrap any single method by a separate UseCase.
But, let's do it in another PR, after the discussion, on how it should be, what do you think, @VEZE ?

Reply:
Let's refactor it later